docs/src/main/sphinx/connector/pinot.rst

```diff
@@ -114,3 +114,19 @@ Pinot                        Trino
 ``STRING_ARRAY``             ``VARCHAR``
 ==========================   ============
 
+.. _pinot-sql-support:
+
+SQL support
+-----------
+
+The connector provides read and write access to data and metadata in
+the Pinot database. In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
+
+* :doc:`/sql/insert`
+* :doc:`/sql/create-table`
+* :doc:`/sql/create-table-as`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/create-schema`
+* :doc:`/sql/drop-schema`
```

Comment:
I think all statements aren't yet supported and they are ongoing in #7164, #7162, #7160
https://github.com/trinodb/trino/blob/master/plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotMetadata.java

Reply:
@Jessie212 You can remove all these statements and use the read-only wording like in Druid